### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by mkitzmann**
image

### CODE REVIEW

File: frontend/src/store/NotificationStore.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+export const useNotificationStore = defineStore('notification', () => {
+  const notifications = reactive<Record<string, Notification>>({});
+
+  const remove = (id: string) => {
+    delete notifications[id];
+  };
+
+  const create = (type: NotificationType, message: string, progress?: number): string => {
```
Comment (sadortun) [inline frontend/src/store/NotificationStore.ts:28]: It make probably more sense to accept an object directly to add flexibility to the params 

```show = ({type: NotificationType, message: string, progress?: number}): string => {



```

Comment (mkitzmann) [reply frontend/src/store/NotificationStore.ts]: Did something similar. Hope you like it

**Comment by mkitzmann** [comment]
Changing progress is not working after routing

**Review by sadortun — CHANGES_REQUESTED** [review]
Is it me or you're missing `'./NotificationStore';` ?

**Comment by mkitzmann** [comment]
> Is it me or you're missing `'./NotificationStore';` ?

Everything was missing, apparently I was in a rush committing ;)

**Review by sadortun — APPROVED** [review]

**Pull request merged**